app/controllers/supervisor/base_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Supervisor::BaseController < ApplicationController
+
+  before_action :must_be_supervisor
```

Comment:
I will have to look into that again. One of the coaches suggested another solution, so it is on the agenda. 
Not only there is cancancan, but there is also this: https://github.com/rails-girls-summer-of-code/rgsoc-teams/blob/master/app/models/user.rb#L52
And there are all kinds of supervisor-checks, like this one in the header: 
https://github.com/rails-girls-summer-of-code/rgsoc-teams/blob/master/app/views/layouts/_header.html.slim#L26
I don't understand yet what the differences are, and when we can or cancan't use cancancan. Thanks for the tip; to be continued.